packages/block-editor/src/components/image-size-control/index.js

```diff
@@ -97,6 +100,7 @@ export default function ImageSizeControl( {
 												scaledWidth
 											)
 										}
+										size="__unstable-large"
```

Comment:
I'm guessing this one was unintentional? `Button` doesn't have a large variant 😄